content/articles/terraform-provider.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: Terraform Provider
+excerpt: How to use DNSimple's Terraform provider to manage your DNS
+categories:
+- Services
+- Terraform and DNSimple
+---
+
+# DNSimple Terraform Provider
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+[Terraform](https://terraform.io) is an Infrastructure as Code (IoC) tool that allows you to build, change, and version
+infrastructure safely and efficiently. With [DNSimple's Terraform Provider](https://registry.terraform.io/providers/dnsimple/dnsimple/latest),
+you can easily manage your DNS infrastructure and set up the required DNS records.
+
+## Configuring the DNSimple provider
+
+```
+provider "dnsimple" {
+    token    = "${var.dnsimple_token}"
+    account  = "${var.dnsimple_account}"
+    sandbox  = true
+    prefetch = false
+}
+```
+
+To start using DNSimple's Terraform Provider, you'll have to configure it.
+
+The DNSimple API v2 token must be provided in the token part of the provider's configuration. You can use either a User or an Account 
+Token. We recommend using the Account token. You can read our 
+[API Access Token](https://support.dnsimple.com/articles/api-access-token/) support article for more information on how to get a token. 
+
+The **account** argument is the account ID associated with the token. For more information on obtaining your 
+account ID, take a look at [Account scope](https://developer.dnsimple.com/v2/#account-scope) in our 
+[developer documentation](http://developer.dnsimple.com).
+
+The **sandbox** argument tells the provider to use the sandbox environment. The token and account ID will vary 
+between the sandbox and our production environment. The sandbox environment is a great place to make sure you have your 
+configuration dialed in before using it in the actual production environment. To learn more about our sandbox 
+environment, read [Sandbox for Testing Calls to the API](https://support.dnsimple.com/articles/sandbox/).
```

Comment:
There's a few more cases similar to this with odd line-breaks.

What did you use to write this, Emacs? :trollface:

```suggestion
The **sandbox** argument tells the provider to use the sandbox environment. The token and account ID will vary between the sandbox and our production environment. The sandbox environment is a great place to make sure you have your configuration dialed in before using it in the actual production environment. To learn more about our sandbox environment, read [Sandbox for Testing Calls to the API](https://support.dnsimple.com/articles/sandbox/).
```

Reply:
> 

Just vim... :trollface: